common/media-type/src/main/java/io/helidon/common/media/type/MediaTypeImpl.java

```diff
@@ -16,10 +16,30 @@
 
 package io.helidon.common.media.type;
 
+import java.util.Optional;
+
 record MediaTypeImpl(String type, String subtype, String text) implements MediaType {
-    static MediaType parse(String fullType) {
+
+    private static final System.Logger LOGGER = System.getLogger(MediaTypeImpl.class.getName());
+
+    static MediaType parse(String fullType, ParserMode parserMode) {
         int slashIndex = fullType.indexOf('/');
         if (slashIndex < 1) {
+            if (parserMode == ParserMode.RELAXED) {
+                Optional<String> maybeRelaxedType = ParserMode.findRelaxedMediaType(fullType);
+                if (maybeRelaxedType.isPresent()) {
+                    String relaxedType = maybeRelaxedType.get();
+                    slashIndex = relaxedType.indexOf('/');
+                    LOGGER.log(System.Logger.Level.WARNING,
```

Comment:
Not a warning material - esp. if we explicitly set  `RELAXED` mode and provide a replacement, it may be a trace or debug statement.

Reply:
+1; TRACE level maybe